script/tasks/diamondEMERGENCYPause.sh

```
@@ -0,0 +1,209 @@
+    echo "Please enter the addresses of all facets that SHOULD NOT be reactivated while unpausing the diamond:"
+    echo "Required format (including brackets and quotes):   ["0x123...", "0xbeb..."] (or press ENTER to reactivate all facets)"
+    read -r BLACKLIST
+    if [[ -z "$BLACKLIST" ]]; then
+    BLACKLIST="[]"
+    fi
+  fi
+
+  # logging for debug purposes
+  echo ""
+  echoDebug "in function diamondEMERGENCYPause"
+  echoDebug "NETWORKS=$NETWORKS"
```

Shellcheck

[warning] Expanding an array without an index only gives the first element.

(SC2128)